Helper/Cart.php

```diff
@@ -1173,14 +1173,19 @@ function ($item) use ($imageHelper, &$totalAmount, &$diff, $storeId, $currencyCo
 
                 // This will override the $_product with the variant product to get the variant image rather than the main product image.
                 try {
-                    $variantProductToGetImage = $this->productRepository->get($item->getSku(), false, $storeId);
+                    // If the cart item is type of bundle product, its SKU is a combination with bundle selections,
+                    // so we need to retrieve the sku of bundle product without bundle selections.
+                    $product_sku = 'bundle' == $item->getProductType()
+                                   ? $_product->getData('sku')
+                                   : $product['sku'];
+                    $variantProductToGetImage = $this->productRepository->get($product_sku, false, $storeId);
```

Comment:
Would `$_product->getData('sku')` work in every case?

Reply:
@danac-gs It does not work for configurable product,  $_product->getData('sku') just return the sku of parent product.